includes/wikia/VariablesBase.php

```diff
@@ -8978,3 +8978,13 @@
  * @var string[] $wgTriviaQuizzesEnabledPages
  */
 $wgTriviaQuizzesEnabledPages = [];
+
+/**
+ * ArticleSaveHooks RabbitMQ configuration.
+ * @see extensions/wikia/TaxonomyArticleHooks
+ * @var array $wgTaxonomyArticleExchange
+ */
+$wgTaxonomyArticleExchange = [
```

Comment:
I wonder if it's better to keep this `TaxonomyArticleHooks.php`. That way we can keep all the extensions configuration in a single place. What do you think?

Reply:
I think we should stick with convention here, as it looks like most extensions that communicate with Rabbit also store their configuration in the global VariablesBase. As we're now also going to have `wgEnableTaxonomyArticles` we'll have _some_ external config in there anyway.